src/dinojump/util/Audio.java

```diff
@@ -0,0 +1,153 @@
+package dinojump.util;
+
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.Clip;
+import javax.sound.sampled.LineUnavailableException;
+import java.io.File;
+import java.io.IOException;
+
+public class Audio
+{
+    //bitte nicht angucken
+    private static Audio instance;
+    private AudioInputStream gameStart;
+    private AudioInputStream gameLoop;
+    private AudioInputStream lobbyLoop;
+    Clip bgMusic;
+    private AudioInputStream jump;
+    private AudioInputStream gameOver;
+
+    private Audio()
+    {
+        try
+        {
+            gameStart = AudioSystem.getAudioInputStream(new File("./resources/audio/music/soundtrack_start.wav").getAbsoluteFile());
+            gameLoop = AudioSystem.getAudioInputStream(new File("./resources/audio/music/soundtrack_loop.wav").getAbsoluteFile());
+            lobbyLoop = AudioSystem.getAudioInputStream(new File("./resources/audio/music/lobby.wav").getAbsoluteFile());
+            jump = AudioSystem.getAudioInputStream(new File("./resources/audio/fx/jump_1.wav"));
+            gameOver = AudioSystem.getAudioInputStream(new File("./resources/audio/fx/gameover2.wav"));
+        }
+        catch (Exception e)
+        {
+            throw new RuntimeException(e);
+        }
+        try
+        {
+            bgMusic = AudioSystem.getClip();
```

Comment:
Can't this get moved into the try block above, just add the exception to the catch clause